modules/articles/client/services/models/article.client.model.js

```diff
@@ -0,0 +1,19 @@
+(function () {
```

Comment:
Ok. I see you removed *articles.client.service*. However, why add it to a *models* folder & rename with *.model.js*?

Also, the name should `ArticleService` to follow John Papa's guide.